config/ncit.iris

```diff
@@ -49,3 +49,4 @@
 +(http://purl.bioontology.org/ontology/npo#NPO_1436):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C47874 Separation Column;Column;Separation Column
 +(http://purl.obolibrary.org/obo/CHEBI_33232):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C63495 Additive
 +(http://livercancer.imbi.uni-heidelberg.de/ccont#CCONT_0000047):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C44374 Diesel Exhaust Particulate
++(http://semanticscience.org/resource/CHEMINF_000025):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C157206 Adsorption
```

Comment:
Adsorption is, I guess, a process. I think the superclass should be http://purl.obolibrary.org/obo/BFO_0000015 ?

Reply:
CHEMINF_000025 is a physical descriptor. Now, I think adsorption could be used as that, but I think, given it's from a thesaurus, the refer to a process. A descriptor if not a process, but a bit of information.

Reply:
Okay, after reviewing the difference between BFO and CHEMINF superclass, I see your point. I will update this PR.